src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/ScheduleController.java

```diff
@@ -95,19 +97,50 @@ public String add( @PathVariable("scheduleid") Long scheduleid,
         primary.setQuarter(lecture_quarter);
         primary.setScheduleid(scheduleid);
         logger.info("primary = " + primary); 
-        scheduleItemRepository.save(primary);
+        // scheduleItemRepository.save(primary);
```

Comment:
Is there a reason for why this is commented out? This removed the ability to add a section's associated lecture to the schedule.

Reply:
We copied and pasted “CourseSchedule” to “ScheduleController.” While testing, it made sense not implement this feature for what we’re doing. Should we still leave it in?

Reply:
Did you have any conflicts with leaving it in? This feature is essential to the operation of a schedule itself, so you should definitely leave it in (or make fixes so you can).

Reply:
The commented line is not essential. It adds a the corresponding lecture time of the section that was added (When you add a section, it will also add the lecture time. The section time is added on scheduleItemRepository.save(scheduleItem); on line 86/88). The commented part is a feature that is a WIP.